scraper/doc/permapeople_columns.md

```diff
@@ -0,0 +1,172 @@
+# Permapeople and PracticalPlants columns
+
+## Merged columns
+
+Total number of merged columns: 15
+
+| PracticalPlants column name | Permapeople name    | New column name       |
+| --------------------------- | ------------------- | --------------------- |
+| binomial_name               | scientific_name     | scientific_name       |
```

Comment:
Isn't this unique_name? I think it is better if we document this in the Rust struct. Or is this for Yvonne and you copy it to Rust later?

Reply:
I created these mapping docs for myself during the merging and before putting them into the struct docs. But I'm currently stuck with Rust and will ask for help in the chat. So I left them for now and can delete afterwards

Reply:
Ok, then it is okay. I probably cannot give you useful feedback if you don't tell me what is finished and what I should review.